### ISSUE TITLE
tests and CI

### ISSUE THREAD

**Issue by stenczelt** [ZimmermanGroup/pyGSM#31]
I noticed that there are no unit tests for the repo yet, though I have written some for my ASE interface in my fork and enabled testing with GitHub actions there. This works very well and GitHub gives us unlimited build minutes on public repos.

Can I merge this into the main repo as well with the ASE interface and its tests? This would enable testing, coverage and CI in your main repo as well!

For now, there is a branch off of the ASE development branch in my fork (https://github.com/stenczelt/pyGSM/tree/actions-tests), I should merge that into the ASE dev branch and that would add it to #29 as well.

**Comment by craldaz** [comment]
Yes, I'd like that. Unfortunately I don't know a lot about testing on GitHub but I'd be interested to learn and implement it here.

**Comment by joshkamm** [comment]
@craldaz I was thinking about setting up a github action and writing a basic test for pyGSM because there are a couple of other issues I've considered trying to fix but I'm scared of breaking other things in the process. Then I saw that this already exists and that your experiment at https://github.com/craldaz/pyGSM has a github action as well.

Are you still hoping to merge that experiment on your account into this main repo?

If I get some time I can try to work on prepping to merge.
